admin-auth/src/test/kotlin/com/epam/drill/admin/auth/UserRepositoryImplTest.kt

```diff
@@ -15,19 +15,234 @@
  */
 package com.epam.drill.admin.auth
 
-import org.mockito.MockitoAnnotations
+import com.epam.drill.admin.auth.entity.UserEntity
+import com.epam.drill.admin.auth.principal.Role
+import com.epam.drill.admin.auth.repository.impl.UserRepositoryImpl
+import com.epam.drill.admin.auth.table.UserTable
+import kotlinx.coroutines.runBlocking
+import org.jetbrains.exposed.sql.*
+import org.jetbrains.exposed.sql.statements.InsertStatement
+import org.jetbrains.exposed.sql.transactions.experimental.newSuspendedTransaction
+import org.jetbrains.exposed.sql.transactions.transaction
 import kotlin.test.*
 
 class UserRepositoryImplTest {
+
+    private val repository = UserRepositoryImpl()
+
     @BeforeTest
     fun setup() {
-        MockitoAnnotations.openMocks(this)
+        Database.connect("jdbc:h2:mem:test;DB_CLOSE_DELAY=-1", "org.h2.Driver", user = "root", password = "")
+        transaction {
+            SchemaUtils.createSchema(Schema("auth"))
+            SchemaUtils.create(UserTable)
+        }
+    }
+
+    @Test
+    fun `given unique username, create must insert user and return id`() = withTransaction {
+        val id = repository.create(
+            UserEntity(
+                username = "uniquename", passwordHash = "hash", role = "USER"
+            )
+        )
+
```

Comment:
add assertion to verify id is not null / contains expected value (positive integer?)

Reply:
id is not nullable